CHANGELOG.md

```diff
@@ -1,3 +1,179 @@
+# 0.28.0 (2019-04-??)
+
+## Language changes
+
+- **(breaking-change)** Enum declaration members can no longer be separated by a space, only by a newline, `;` or `,`, the latter being deprecated and reformatted to a newline. ([#7607](https://github.com/crystal-lang/crystal/pull/7607), [#7618](https://github.com/crystal-lang/crystal/pull/7618), thanks @asterite, and @j8r)
+- Add begin-less and end-less ranges: `array[5..]`.  ([#7179](https://github.com/crystal-lang/crystal/pull/7179), thanks @asterite)
+- Add `offsetof(Type, @ivar)` expression. ([#7589](https://github.com/crystal-lang/crystal/pull/7589), thanks @malte-v)
+
+### Macros
+
+- Add `Type#annotations` to list all annotations and not just the last of each kind. ([#7326](https://github.com/crystal-lang/crystal/pull/7326), thanks @Blacksmoke16)
+- Add `ArrayLiteral#sort_by` macro method. ([#3947](https://github.com/crystal-lang/crystal/pull/3947), thanks @jreinert)
+
+## Standard library
+
+- **(breaking-change)** Allow creating `None` enum flag with `Enum.from_value`. ([#6516](https://github.com/crystal-lang/crystal/pull/6516), thanks @bew)
+- **(breaking-change)** Add deprecation message to `PartialComparable`. Its behaviour has been fully integrated into `Comparable`. ([#7664](https://github.com/crystal-lang/crystal/pull/7664), thanks @straight-shoota)
+- **(performance)** Optimize dwarf line numbers decoding. ([#7413](https://github.com/crystal-lang/crystal/pull/7413), thanks @asterite)
```

Comment:
I think performance improvements are good to be aware of. One might change an implementation based on it. Not in this case, but that is the reason performance is highlighted.

On the other hand, refactors goes after additions since those would not matter to the programmer.

Reply:
I don't understand why they're more important than additions, and they've never been highlighted before. Please don't do this.

Reply:
We can review the order for next release. 

The order of importance is subjective. I think bugfixes are more important than additions because they can have an impact on things you assume/workaround. Same criteria can work with performance.

Additions are great but will never require an action to upgrade your code.

Reply:
The ordering doesn't matter much to me, but highlighting one specific category in bold is another thing.